kinto_attachment/tests/__init__.py

```diff
@@ -99,9 +99,21 @@ def tearDown(self):
 class BaseWebTestS3(BaseWebTest):
     config = 'config/s3.ini'
 
-    def upload(self, *args, **kwargs):
-        # XXX: use moto_server instead.
-        patch = mock.patch('pyramid_storage.s3.S3FileStorage.save_file',
-                           return_value='upload.jpg')
-        with patch.start():
-            return super(BaseWebTestS3, self).upload(*args, **kwargs)
+    def __init__(self, *args, **kwargs):
+        self._bucket_created = False
+        super(BaseWebTestS3, self).__init__(*args, **kwargs)
+
+    def make_app(self):
+        app = super(BaseWebTestS3, self).make_app()
+
+        # Create bucket if necessary
```

Comment:
We should specify that we are talking about the S3 bucket and not the Kinto one.